lib/internal/worker/js_transferable.js

```diff
@@ -49,8 +48,8 @@ module.exports = {
   makeTransferable,
   setup,
   JSTransferable,
-  kClone: messaging_clone_symbol,
-  kDeserialize: messaging_deserialize_symbol,
+  kClone: SymbolFor('kClone'),
+  kDeserialize: SymbolFor('kDeserialize'),
```

Comment:
Same here for the symbols. Since these are using the global symbol namespace now, they need to use the `nodejs.` prefix.

Reply:
done in https://github.com/nodejs/node/pull/41148/commits/1a8395eba12658303f0f1cf4e81f2bcd8686978f